cli/internal/taskhash/taskhash.go

```diff
@@ -204,18 +205,18 @@ func (th *Tracker) CalculateFileHashes(
 
 	for i := 0; i < workerCount; i++ {
 		hashErrs.Go(func() error {
-			for packageFileSpec := range hashQueue {
-				pkg, ok := workspaceInfos.PackageJSONs[packageFileSpec.pkg]
+			for PackageFileSpec := range hashQueue {
```

Comment:
I think the find/replace went wrong here and we now have a loop variable with the same identifier as the struct which makes it hard to read.

Reply:
reverted, good catch